shell/platform/embedder/embedder.h

```diff
@@ -757,6 +757,9 @@ typedef struct {
   /// The queue family index of the VkQueue supplied in the next field.
   uint32_t queue_family_index;
   /// VkQueue handle.
+  /// The queue should not be used without protection from a mutex to make sure
+  /// it is not used simultaneously with other threads. That mutex should match
+  /// the one injected via the |get_instance_proc_address_callback|.
```

Comment:
Should we link to https://github.com/flutter/flutter/issues/134573 (bug about fixing this behavior)?

Reply:
done